### PR TITLE
fix (#94): don't initialize db by hand

### LINKED ISSUE #94: org.h2.jdbc.JdbcSQLException: Unique index or primary key violation: "PRIMARY_KEY_A ON PUBLIC.BOT_USER(USERNAME) VALUES ('admin', 1)
## Issue

A SQL error is reported when we try to authenticate the user from the UI

```Caused by: org.h2.jdbc.JdbcSQLException: Unique index or primary key violation: "PRIMARY_KEY_A ON PUBLIC.BOT_USER(USERNAME) VALUES ('admin', 1)"; SQL statement:

INSERT INTO bot_user (id, username, password, role) VALUES (1, 'admin', 'admin', 'admin') [23505-197]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:357)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.index.BaseIndex.getDuplicateKeyException(BaseIndex.java:101)
	at org.h2.mvstore.db.MVSecondaryIndex.requireUnique(MVSecondaryIndex.java:236)
	at org.h2.mvstore.db.MVSecondaryIndex.add(MVSecondaryIndex.java:202)
	at org.h2.mvstore.db.MVTable.addRow(MVTable.java:732)
	at org.h2.command.dml.Insert.insertRows(Insert.java:182)
	at org.h2.command.dml.Insert.update(Insert.java:134)
	at org.h2.command.dml.RunScriptCommand.execute(RunScriptCommand.java:79)
	at org.h2.command.dml.RunScriptCommand.update(RunScriptCommand.java:58)
	at org.h2.command.CommandContainer.update(CommandContainer.java:102)
	at org.h2.command.Command.executeUpdate(Command.java:261)
	at org.h2.engine.Engine.openSession(Engine.java:227)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:157)
	at org.h2.engine.Engine.createSession(Engine.java:140)
	at org.h2.engine.Engine.createSession(Engine.java:28)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:351)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:124)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:103)
	at org.h2.Driver.connect(Driver.java:69)
	at io.agroal.pool.ConnectionFactory.createConnection(ConnectionFactory.java:200)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:419)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:401)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at io.agroal.pool.util.PriorityScheduledExecutor.beforeExecute(PriorityScheduledExecutor.java:65)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1126)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
```

### PR DESCRIPTION
**Pull request by iocanel**
Resolves: #94

### CODE REVIEW

File: src/main/resources/application.properties [hunk 1]
```diff
@@ -76,7 +76,7 @@ quarkus.datasource.min-size=3
 quarkus.datasource.max-size=13
 quarkus.hibernate-orm.database.generation=drop-and-create
 
-%dev.quarkus.datasource.url=jdbc:h2:mem:snowdrop-bot;DB_CLOSE_DELAY=-1;DB_CLOSE_DELAY=-1;INIT=RUNSCRIPT FROM 'classpath:import.sql'
+%dev.quarkus.datasource.url=jdbc:h2:mem:snowdrop-bot;DB_CLOSE_DELAY=-1;DB_CLOSE_DELAY=-1
```
Comment (cmoulliard) [inline src/main/resources/application.properties:79]: The `DB_CLOSE_DELAY` is defined `twice`. To be fixed ;-)

Comment (cmoulliard) [reply src/main/resources/application.properties:79]: If the import.sql script is not injected, how will the DB table created and user admin/admin created ?

**Comment by jacobdotcosta** [comment]
If we leave in prod the `quarkus.hibernate-orm.database.generation=drop-and-create` it will solve the problem... Is brute force but for now it might do the trick. Moreover, with this approach, we shouldn't even need a persistent volume... Maybe later we can have a decent database and stop having problems such as #100 .

**Comment by cmoulliard** [comment]
I think that this PR should be closed @iocanel @aureamunoz

**Comment by aureamunoz** [comment]
Yes, I agree

**Pull request closed without merging**